components/camel-rabbitmq/src/main/java/org/apache/camel/component/rabbitmq/RabbitMQEndpoint.java

```diff
@@ -34,26 +26,11 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.TimeoutException;
 
-import javax.net.ssl.TrustManager;
-
-import com.rabbitmq.client.AMQP;
-import com.rabbitmq.client.Address;
+import com.rabbitmq.client.*;
 import com.rabbitmq.client.Channel;
-import com.rabbitmq.client.Connection;
-import com.rabbitmq.client.ConnectionFactory;
-import com.rabbitmq.client.Envelope;
-import com.rabbitmq.client.LongString;
-
 import org.apache.camel.Consumer;
-import org.apache.camel.Exchange;
-import org.apache.camel.Message;
-import org.apache.camel.NoTypeConversionAvailableException;
-import org.apache.camel.Processor;
-import org.apache.camel.Producer;
-import org.apache.camel.RuntimeCamelException;
-import org.apache.camel.TypeConversionException;
+import org.apache.camel.*;
```

Comment:
please do not do wildcard imports. See building with checkstyle here to make the code comply: http://camel.apache.org/building